qiskit/backends/_qeremote.py

```diff
@@ -0,0 +1,112 @@
+import time
+import logging
+from qiskit.backends._basebackend import BaseBackend
+from qiskit.backends._backendutils import get_backend_configuration
+from qiskit import _openquantumcompiler as openquantumcompiler
+from qiskit._result import Result
+from IBMQuantumExperience.IBMQuantumExperience import IBMQuantumExperience
+
+logger = logging.getLogger(__name__)
+
+class QeRemote(BaseBackend):
+    def __init__(self, configuration=None):
+        """Initialize remote backend for IBM Quantum Experience.
+
+        Args:
+
+        """
+        if configuration is None:
+            self._configuration = get_backend_configuration(backend_name)
+        else:
+            self._configuration = configuration
+        self._configuration['local'] = False
+
+    def run(self, q_job):
```

Comment:
Ideally, subclasses' methods should have the same signatures as the superclass methods they override - in this case, `BaseBackend` currently has `def run(self)` (without the `q_job` parameter). Would it be possible to:
* if all the backends now expect a `q_job` in `run()` (which it seems to be the case), change `BaseBackend.run()` signature requiring that parameter?
* if not, perhaps an approach based on `**kwargs` (plus documenting properly) would help - although should be used with care.

Reply:
done